### PR TITLE
Backport/rfe 748

### PR DESCRIPTION
**Pull request by shpran**
Backport: https://github.com/oat-sa/extension-tao-ltideliveryprovider/pull/362

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -68,7 +68,7 @@
 		"oat-sa/extension-tao-delivery-rdf" : ">=14.0.0",
 		"oat-sa/extension-tao-lti" : ">=15.11.0",
 		"oat-sa/extension-tao-outcomeui" : ">=10.0.0",
-		"oat-sa/extension-tao-testqti" : ">=42.2.0",
+		"oat-sa/extension-tao-testqti" : "dev-backport/rfe-748/45.0.6.2 as 45.0.6.2",
```
Comment (shpran) [inline composer.json:71]: - [ ] Change before merge

**Review by hectoras — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (hectoras) [reply composer.json]: 28ac471d390a7e14a084fe8e47b3f4df3959c831

File: composer.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
 		"oat-sa/extension-tao-delivery-rdf" : ">=14.0.0",
 		"oat-sa/extension-tao-lti" : ">=15.11.0",
 		"oat-sa/extension-tao-outcomeui" : ">=10.0.0",
-		"oat-sa/extension-tao-testqti" : "dev-backport/rfe-748/45.0.6.2 as 45.0.6.2",
+		"oat-sa/extension-tao-testqti" : ">=45.0.6.3",
```
Comment (hectoras) [inline composer.json:71]: Points to https://github.com/oat-sa/extension-tao-testqti/pull/2433

**Comment by hectoras** [comment]
Released as [v12.15.6](https://github.com/oat-sa/extension-tao-ltideliveryprovider/releases/tag/v12.15.6).

**Pull request merged**